AdyenActions/Components/SDK/TwintSDKActionComponent.swift

```diff
@@ -1,130 +1,130 @@
-        ///   - localizationParameters: The localization parameters, leave it nil to use the default parameters.
-        public init(style: AwaitComponentStyle = .init(),
-                    localizationParameters: LocalizationParameters? = nil) {
-            self.style = style
-            self.localizationParameters = localizationParameters
+            /// Initializes an instance of `Configuration`
+            ///
+            /// - Parameters:
+            ///   - style: The Component UI style.
+            ///   - localizationParameters: The localization parameters, leave it nil to use the default parameters.
+            public init(style: AwaitComponentStyle = .init(),
+                        localizationParameters: LocalizationParameters? = nil) {
+                self.style = style
+                self.localizationParameters = localizationParameters
+            }
         }
-    }
 
-    /// The twint component configurations.
-    public var configuration: Configuration
-
-    /// Initializes the `TwintSDKActionComponent`.
-    ///
-    /// - Parameter context: The context object for this component.
-    /// - Parameter configuration: The TwintSDK component configurations.
-    public init(context: AdyenContext,
-                configuration: Configuration = .init()) {
-        self.context = context
-        self.configuration = configuration
-    }
+        /// The twint component configurations.
+        public var configuration: Configuration
 
-    /// Handles TwintSDK action.
-    ///
-    /// - Parameter action: The Twint SDK action object.
-    public func handle(_ action: TwintSDKAction) {
-        AdyenAssertion.assert(message: "presentationDelegate is nil", condition: presentationDelegate == nil)
-        Twint.fetchInstalledAppConfigurations { [weak self] installedApps in
-            if installedApps?.count == 0 {
-                self?.handleShowError("No or an outdated version of TWINT is installed on this device. Please update or install the TWINT app." )
-            } else if installedApps?.count == 1 {
-                guard let installedApps = installedApps else {
-                    return
+        /// Initializes the `TwintSDKActionComponent`.
+        ///
+        /// - Parameter context: The context object for this component.
+        /// - Parameter configuration: The TwintSDK component configurations.
+        public init(context: AdyenContext,
+                    configuration: Configuration = .init()) {
+            self.context = context
+            self.configuration = configuration
+        }
+
+        /// Handles TwintSDK action.
+        ///
+        /// - Parameter action: The Twint SDK action object.
+        public func handle(_ action: TwintSDKAction) {
+            AdyenAssertion.assert(message: "presentationDelegate is nil", condition: presentationDelegate == nil)
+            Twint.fetchInstalledAppConfigurations { [weak self] installedApps in
```

Comment:
guard self since there are multiple `self?` usages

Reply:
Added it :)